src/sagemaker/tensorflow/README.rst

```diff
@@ -460,175 +144,90 @@ you can specify these as keyword arguments.
    framework_version and py_version. Refer to: `SageMaker TensorFlow Docker Containers
    <#sagemaker-tensorflow-docker-containers>`_ for details on what the Official images support
    and where to find the source code to build your custom image.
+- ``script_mode (bool)`` Wether to use Script Mode or not. Setting ``py_version`` to ``py3`` overrides it.
+- ``model_dir (str)`` S3 location where checkpoint data will saved and restored. If not specified a S3 location will
```

Comment:
Location where model data, checkpoint data, and TensorBoard checkpoints should be saved during training.

This argument is typically automatically set by SageMaker with the values:

- single training: model_dir = /opt/ml/model, a special folder in the container that will be persisted after the end of training, see https://docs.aws.amazon.com/sagemaker/latest/dg/your-algorithms-training-algo.html#your-algorithms-training-algo-envvariables for more information

- distributed_training: a S3 location shared between all containers ...

Reply:
Add links to example notebooks or code samples showing this.